apps/arweave/src/ar_kv_sup.erl

```diff
@@ -21,5 +21,5 @@ start_link() ->
 %% ===================================================================
 
 init([]) ->
-	ets:new(ar_kv, [set, public, named_table]),
+	ets:new(ar_kv, [set, public, named_table, {keypos, 2}]),
```

Comment:
Usually it is better to keep the ETS calls in one process, but in this case the supervisor owns the table to ensure it will not be dropped in case of `ar_kv` process termination. 
I will fix this code so that it will be readable.